apps/financial-aid/web-osk/src/components/FormLayout/FormLayout.tsx

```diff
@@ -46,7 +51,7 @@ const FormLayout: React.FC<PageProps> = ({
       background="purple100"
       className={styles.processContainer}
     >
-      {isAuthenticated ? (
+      {isAuthenticated && !user?.hasAppliedForPeriod && (
```

Comment:
What is displayed on this page if the user is undefined? 

Wondering if we should add a check above and have a special case if that happens, something like: 'It seems your are not logged in` and then a button to login or whatever? 

Also could we change ! to === false? 🙏